### DISCUSSION TITLE
Url Params as arrays

### DISCUSSION THREAD

**Discussion by kand617** [mockoon/mockoon#560] [Help] [answered]
**Is your feature request related to a problem? Please describe.**
I have a path like `something/:ids` where `ids` can be a array. E.g `something/1,2,3
`
**Describe the solution you'd like**
I would like to be able to generate a response like 
```
{
   "1":{},
   "2": {},
   "3": {}
}
```

**Describe alternatives you've considered**
I looked at handler bars, but seems like you need to register custom helpers. 
I could not find a way to register custom function in mockoon...

**Additional context**

**Comment by 255kb** [comment, marked as answer]
Mockoon uses ExpressJS which do not parse the URL params as array or anything similar. 
However, you can achieve what you want with a combination of `each`, `split` and `urlParam`:

```handlebars
{
  {{#each (split (urlParam 'list') ',')}}  
  {"{{this}}": ""}{{#unless @last}},{{/unless}}  
  {{/each}}
}
```

`each` and `unless` are from [Handlebars](https://handlebarsjs.com/guide/builtin-helpers.html#each) while [`split`](https://mockoon.com/docs/latest/templating/mockoon-helpers/#split) and [`urlParam`](https://mockoon.com/docs/latest/templating/mockoon-request-helpers/#urlparam) are Mockoon specific helpers.
